api/pkg/cli/hub/get_resource.go

```diff
@@ -34,8 +34,18 @@ type ResourceOption struct {
 	PipelineVersion string
 }
 
+type ResourceResult interface {
+	RawURL() (string, error)
+	Manifest() ([]byte, error)
+	Resource() (interface{}, error)
+	ResourceYaml() (string, error)
+	ResourceVersion() (string, error)
+	MinPipelinesVersion() (string, error)
+	UnmarshalData() error
+}
+
 // ResourceResult defines API response
```

Comment:
Could you please update the comment please

Reply:
Oh good catch! Updated